From 14c6e632ada4ae0f13bd1397d4163c006272a8a0 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Wed, 9 Dec 2020 14:34:27 +0800
Subject: [PATCH] ARM: rockchip_defconfig: enable CONFIG_INCREMENTAL_FS by
 default

Test:
    CtsIncrementalInstallHostTestCases
    android.incrementalinstall.cts.IncrementalFeatureTest

This config will enable the Incremental File System, which allows
Android OS to receive streamed APKs over adb.

Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
Change-Id: I0d903f95ed8c1b1765c36a0bd7fde25695138e5c
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index f4431f605752..289327583574 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -640,6 +640,7 @@ CONFIG_QUOTA_NETLINK_INTERFACE=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
 CONFIG_OVERLAY_FS=y
+CONFIG_INCREMENTAL_FS=y
 CONFIG_ISO9660_FS=y
 CONFIG_JOLIET=y
 CONFIG_ZISOFS=y
-- 
2.35.3

